README.md

```diff
@@ -166,6 +166,91 @@ optional `--wait --timeout 15m` parameters in a deployment pipeline to see if th
 To deploy Magento to different environments (develop, staging, production) it is recommended to create a `values_*.yaml`
 for each environment and tune the resource limits and configuration values of the services.
 
+## Docker Desktop Example
+The file `values_docker.yaml` is a shorter version of `values.yaml` which contains all the changes necessary 
+to execute the chart locally on your machine and create a local cluster with the help of [Docker Desktop](https://www.docker.com/products/docker-desktop).
+The following steps will give a guide on how to achieve that:
+
+### Prerequisites
+- Install *Docker Desktop* locally (Tested with 4.10.1) and enable kubernetes in settings.
+It is also recommended to increase the available resources of Docker Desktop to 4 CPU cores and 8 GB of RAM to ensure a smooth operation. 
+Alternatively you may install *kubectl* separately, however that may be a more error-prone approach.
+- Clone this repository locally on your computer.
+
+### Step 1
+Generate a `kube-config` file which will be used for the helm deployment. Make sure to generate it
+in the same folder as the git repo. Quickest way to do so is:
+
+```
+kubectl config view --raw > <name-of-your-file>
+```
+
+### Step 2
+Have a helm environment set up. We recommend using a docker container for that in order to facilitate the usage of Helm
+and not have to worry about dependencies. However it is also possible to have a native Helm installation for our purposes.
+You may set up the docker container using the following command:
+
+```
+docker run -ti --entrypoint= -v $(pwd):/apps alpine/helm:latest sh
+```
+
+When using the native Helm installation bear in mind that the Helm version needs to be 3.9.x
+
+### Step 3
+Next we want to deploy the helm chart, but before we can do that we will need to pull the dependencies of our chart.
+
+```
+helm dependency update #pulls all the other charts that our chart uses
+helm upgrade -i --kubeconfig <name-of-your-kubeconfig-flie> -f values_docker.yaml --create-namespace -n <your-namespace> magento .
+```
+
+### Step 3.5
+Check the progress of the deployment in the host machine, *not in your docker container*
+
+```
+kubectl get pods -n <your-namespace> -w
+```
+
+### Step 4
+Once the deployment has fully finished all that is missing is an ingress-nginx, which we deploy through Helm as well.
+In the previously created docker container run the following command:
+
+```
+helm upgrade --kubeconfig kube-config --install ingress-nginx ingress-nginx --repo https://kubernetes.github.io/ingress-nginx --namespace ingress-nginx --create-namespace
+```
+
+The command above will deploy a ingress-nginx pod that has access to the `/etc/hosts file`.
```

Comment:
@phoenix-henri "For the Magento installation the domain name "magento.local" is predefined in `values_docker.yaml`. Ensure the domain points to your local K8S cluster, so the Ingress can forward the traffic to the Magento instance:"